core/app/models/spree/address.rb

```diff
@@ -29,6 +29,9 @@ class Address < Spree::Base
 
     validate :state_validate, :postal_code_validate
 
+    delegate :name, :iso3, to: :country
```

Comment:
@VygovskySergey you can also add `prefix: :true`  so it will be already accessible via `country_iso3`

This is a must for `name` as it's kinda ambiguous

Reply:
@damianlegawiec Yeah, nice idea.